docs/tutorials/1. Getting started.md

```diff
@@ -17,7 +17,6 @@ You can also find the latest releases on [Github releases](https://github.com/MT
 <script src="https://cdn.jsdelivr.net/npm/essentia.js@0.1.1/dist/essentia.js-plot.js"></script>
```

Comment:
We could potentially remove the version number from this link to make it generic? Eg: ```"https://cdn.jsdelivr.net/npm/essentia.js@<version>/dist/essentia.js-*.js"```